doc/tor.1.txt

```diff
@@ -2605,6 +2605,16 @@ Relays publish most statistics in a document called the
 extra-info document. The following options affect the different
 types of statistics that Tor relays collect and publish:
 
+[[BandwidthStatistics]] **BandwidthStatistics** **0**|**1**|**auto**::
+    Relays and bridges only.
+    When this option is enabled, Tor collects statistics about bandwidth
+    used each 24 hours for the last 5 days (if available). Reports
+    bandwidth statistics for OR and directory reads and writes,
+    and writes them to disk every 24 hours. Enables relay and bridge
+    operators to monitor bandwidth used by Tor. If ExtraInfoStatistics
+    and the corresponding consensus parameter is enabled, it will be
+    published as part of the extra-info document. (Default: auto)
```

Comment:
This man page entry doesn't answer these questions:
* What happens to statistics collection if the BandwidthStatistics option is auto ?
* Are statistics published when the BandwidthStatistics option is 1 ?
* Are statistics published when the BandwidthStatistics option is 0 ?
* Are statistics published when the BandwidthStatistics consensus parameter is auto?

Please rewrite the consensus parameter part of the man page entry. Use another consensus parameter man page entry as a template:
https://github.com/torproject/tor/blob/master/doc/tor.1.txt#L2766